index.md

```diff
@@ -1,3 +1,6 @@
 ---
 layout: default
 ---
```

Comment:
# This is an <h1> header, which is the largest
## This is an <h2> header
###### This is an <h6> header, which is the smallest